actionmailer/lib/action_mailer/base.rb

```diff
@@ -84,9 +84,9 @@ module ActionMailer
   # = Mailer views
   #
   # Like Action Controller, each mailer class has a corresponding view directory in which each
-  # method of the class looks for a template with its name.
+  # method of the class looks for a template with it's name.
   #
-  # To define a template to be used with a mailing, create an <tt>.erb</tt> file with the same
+  # To define a template to be used with a mail, create an <tt>.erb</tt> file with the same
```

Comment:
Except this change, can you revert all others, they are correct as is right now.